backend/jobsearch/src/main/kotlin/com/jobsearch/service/CvService.kt

```diff
@@ -20,39 +21,57 @@ class CvService(
     @Transactional
     fun createCv(cvDTO: CvRequestDTO): CvResponseDTO {
 
+        // Setting basic attributes
```

Comment:
it seems like that method contains a lot of logic to convert `cvDTO` to `cv`
Please create `CvMapper` and move the all logic related to transforming `cvDTO` to `cv`